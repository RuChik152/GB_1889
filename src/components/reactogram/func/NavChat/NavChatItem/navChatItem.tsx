import React, { FC } from 'react';
import style from './navChatItem.module.scss';

interface ChatItemProps {
    link: string;
    title: string;
    key: number;
}

export const NavChatItem: FC<ChatItemProps> = ({ link, title} ) => {

    return (<li className={style.list__item} data- >
                <a className={style.item__link} href = {link} >{title}</a>
            </li>);

}