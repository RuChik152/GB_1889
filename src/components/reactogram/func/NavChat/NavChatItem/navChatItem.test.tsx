import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import {NavChatItem} from "./navChatItem";


describe('NavChatItem', () => {
    it('Render', ()=> {
        render(<NavChatItem link={'#'} title={'Название'} key={1} />)
    })

    it('Is Element', () => {
        render(<NavChatItem link={'#'} title={'Название'} key={1} />)
        const link = screen.getByText('Название');
        expect(link).toBeInTheDocument();
    })

    it('Snapshot', () => {
        const {asFragment} = render(<NavChatItem link={'#'} title={'Название'} key={1} />)
        expect(asFragment()).toMatchSnapshot();
    })
})