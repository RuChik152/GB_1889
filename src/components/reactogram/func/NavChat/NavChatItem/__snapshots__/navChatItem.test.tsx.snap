// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavChatItem Snapshot 1`] = `
<DocumentFragment>
  <li
    class="list__item"
    data-="true"
  >
    <a
      href="#"
    >
      Название
    </a>
  </li>
</DocumentFragment>
`;
